benchmarks/plots-generators/generate-flatten-merge-flow-plots.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
```

Comment:
Let's use the same name for scripts and benchmarks; e.g. `generate_plots_flow_flatten_merge.py` (see the comment below about renaming the benchmark)